src/components/dialogs/create-contingency-list-dialog.js

```diff
@@ -234,18 +235,7 @@ export const CreateContingencyListDialog = ({ open, onClose }) => {
     };
 
     const handleCriteriaBasedFilterCreation = (filter) => {
-        currentCriteriaBasedFilter.current = {};
-        currentCriteriaBasedFilter.current.id = filter.id;
-        currentCriteriaBasedFilter.current.equipmentType =
```

Comment:
You can use here
Object.assign(currentCriteriaBasedFilter.current, filter.equipmentFilterForm);